polaris-react/src/components/Text/README.md

```diff
@@ -18,9 +18,6 @@ keywords:
   - semibold
```

Comment:
@yurm04 can we remove this readme file? I don't think we're using it and the content _should_ be the same as what's in text.md within polaris.shopify.com

Reply:
yeah i think you're good to delete 👍